src/main/java/com/jalasoft/webservice/model/VideoConvert.java

```diff
@@ -1,44 +1,60 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
 package com.jalasoft.webservice.model;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
 import ws.schild.jave.*;
+
 import java.io.File;
+import java.io.IOException;
 
 /**
- *
- * @project WebService feature(VideoConvert)
- * @author Isaac Vasquez on 09/23/2019
+ * VideoConvert implements interface IConvert for using the conversion
  */
-@RestController
-@RequestMapping("/api/v1.0/videoConv")
-public class VideoConvert {
-
-    @GetMapping
-    public String ConverterVideo () throws EncoderException {
+public class VideoConvert implements IConvert {
```

Comment:
What happen with the tabulation in this line?

Reply:
Solved. It was a mistake in my editor.